doc/python/migrate-to-maplibre.md

```diff
@@ -0,0 +1,71 @@
+---
+jupyter:
+  jupytext:
+    notebook_metadata_filter: all
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.16.1
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+  language_info:
+    codemirror_mode:
+      name: ipython
+      version: 3
+    file_extension: .py
+    mimetype: text/x-python
+    name: python
+    nbconvert_exporter: python
+    pygments_lexer: ipython3
+    version: 3.10.11
+  plotly:
+    description: Migrating from Mapbox traces to MapLibre traces.
+    display_as: basic
+    language: python
+    layout: base
+    name: MapLibre Migration
+    order: 1
+    page_type: example_index
+    permalink: python/mapbox-to-maplibre/
+    redirect_from: python/mapbox-migration/
+    thumbnail: thumbnail/line-and-scatter.jpg
+---
+
+## Migrating from Mapbox traces to MapLibre traces
+
+With the release of Plotly.py v5.24.0, we are introducing a new set of trace types for maps with tile underlays, including from Plotly Express:
+- `px.scatter_map`
+- `px.line_map`
+- `px.choropleth_map`
+- `px.density_map`
+
+as well as Plotly Graph Objects:
+- `go.Choroplethmap`
+- `go.Scattermap`
+- `go.Densitymap`
+
+These traces replace the existing Mapbox traces, `px.scatter_mapbox`, `px.line_mapbox`, etc., but use [MapLibre](https://maplibre.org) as the map renderer rather than Mapbox.
+
+When switching to the new traces, keep an eye out for improved rendering performance, WebGL2 support, and over time, improved features in the Plotly map traces inherited from the MapLibre renderer, including projection support, globe views, terrain support, and support for modern mapping standards.
+
+You can learn  more about the motivations for this change in our [announcement post](https://plotly.com/blog/plotly-is-switching-to-maplibre/).
+
+As a result of removing Mapbox as the rendering engine, we're also removing the Mapbox branding from these trace names. This means that migrating from Mapbox traces to MapLibre traces will require some code changes in your projects.
```

Comment:
By redirecting mapbox to map, perhaps we would be able to handle this change (in most cases) without users have to do the change in their code.
```suggestion
As a result of removing Mapbox as the rendering engine, we're also removing the Mapbox branding from these trace names. This means that migrating from Mapbox traces to MapLibre traces may require some code changes in your projects.
```

Reply:
I think you're right on this, and chatted with @alexcjohnson about how we can handle this (maybe by leveraging good old `clean_layout` which was/is used for mapping `is_category`/`is_log` to `access_types`).

That said for the minor release, if users want to take advantage of MapLibre, they _do_ need to change their code, so this prose is accurate today even if we might modify it in a future major release.